docker/girder/Dockerfile

```diff
@@ -1,32 +1,39 @@
-FROM girder/girder:2.x-maintenance-py3
+FROM girder/girder:latest-py3
 LABEL maintainer="patrick.oleary@kitware.com" \
       version="1.0"
 
 # Install cumulus
 RUN git clone https://github.com/Kitware/cumulus.git /cumulus && \
   cd /cumulus && \
-  git checkout 0.9-maintenance && \
+  git checkout master && \
   cd / && \
   pip install --no-cache-dir -r /cumulus/requirements.txt && \
   pip install --no-cache-dir -e /cumulus && \
-  girder-install plugin /cumulus/girder/cumulus && \
-  girder-install plugin /cumulus/girder/sftp && \
-  girder-install plugin /cumulus/girder/taskflow
+  pip install --no-cache-dir -e /cumulus/girder/cumulus && \
+  pip install --no-cache-dir -e /cumulus/girder/sftp && \
+  pip install --no-cache-dir -e /cumulus/girder/taskflow
+
+# TEMP cumulus fix
+COPY ./cumulus/cumulus/girderclient.py /cumulus/cumulus/girderclient.py
```

Comment:
This line is part of your development workflow, correct?  Since there's normally no `cumulus` directory at the top level of this repo...

Reply:
oops, missed that one. Fixed.